### DISCUSSION TITLE
Now that Android Sdk is a must... What? Where?

### DISCUSSION THREAD

**Discussion by avicbk** [airsdk/Adobe-Runtime-Support#1181] [Q&A]
Hello,
First thing, I apologize for the Noob question :( 
I'm back to business making android apps using Air Sdk, after about 2 years off...
Catching up with compiling an .AAB file through ADT and getting error that I have to define an Android Sdk path.
I have never needed it before and now I see it is a must.
I can't find a good explanation telling what exactly should I download, the minimum for it to work, and if that is another thing I have to worry about keeping up with its version updates...
Can someone give me a hand here?
Thanks in advance :)

**Comment by marchbold** [comment]
(Posted on starling forum, but just incase) 

You can have a look at this tutorial, still needs some work but should help you get started:

https://airsdk.dev/docs/tutorials/platform/android/packaging-android-app-bundles

Let me know if you have any questions.

**Reply by avicbk** [reply]
Thanks a lot.
Is there another way to get Android Sdk essential files without downloading Android Studio? It takes around 1GB Storage space or more...
I've downloaded the Basic Android Command Line Tools which takes around 100MB, but I'm not sure it is relevant, and if it is, where is the Android Sdk folder at.

**Reply by marchbold** [reply]
If you download the tools only you have to use the sdkmanager to download the sdk as well. Maybe have a look at this:

https://guides.codepath.com/android/installing-android-sdk-tools

I haven't tried it, as I always have android studio installed (ane development)...
